CHANGELOG.md

```diff
@@ -1,9 +1,32 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.12.0] (date)
+
+#### Added
+
+- Support for local copybooks
+- Dynamic registration for watchers
+
+#### Changed
+
+- Adjust the info message about missing copybooks
+- Reduce the number of error/informational messages
+- Inform user about wrong credentials and stop sending other MF requests
+- Updates to README
+
+#### Fixed
+
+- Create a .gitignore file if it doesn't exist
+- Fix glob pattern for dynamic registration
+- Fix issue with indirect overwrite settings conf for DSN paths
+- Fix issues related to unrecognised copybook extensions
+- Fix representation of settings.json in Theia
+- Use sync APIs to create a folder and write to a file to avoid unexpected behavior
```

Comment:
Previously have been adding PR links, I think we should keep it.

Reply:
We were. I was told by @zimlu02 to stop doing this as of v0.11.0 and to use the added/fixed/changed format, not listing every PR or commit and just summarising main ones, without PR numbers.
We need to agree on a format for the changelog going forward, we can't just keep changing our minds about it. Let's talk about it at the standup today.